onnxmltools/convert/keras/convert.py

```diff
@@ -31,13 +35,23 @@ def convert(model, name=None, initial_types=None, doc_string='', targeted_onnx=o
     produced model. If ONNXMLTools cannot find a compatible ONNX python package, an error may be thrown.
     :return: An ONNX model (type: ModelProto) which is equivalent to the input Keras model
     '''
-    topology = parse_keras(model, initial_types, targeted_onnx)
+    try:
```

Comment:
I am not sure why try-finally block is used. Couldn't the same functionality be done with if-else? We should also note that the conversion becomes not thread-safe now.